src/rollup/form/construct-form-state.js

```diff
@@ -0,0 +1,62 @@
+/** @module pathfora/form/construct-form-state */
+
+// dom
+import document from '../dom/document';
+
+/**
+ * Setup html for success or error state of a form module
+ *
+ * @exports constructFormState
+ * @params {object} widget
+ * @params {object} config
+ * @params {string} name
+ */
+export default function constructFormState (config, widget, name) {
+  var obj, defaultHeadline, defaultMsg;
+
+  switch (name) {
+  case 'success':
+    obj = config.formStates.success;
+    defaultMsg = 'We have received your submission.';
+    defaultHeadline = 'Thank You';
+    break;
+  case 'error':
+    obj = config.formStates.error;
+    defaultMsg = 'There was an error receiving with your submission.';
+    defaultHeadline = 'Error';
+    break;
+  default:
+    throw new Error('Unrecognized formState: ' + name);
+  }
+
+  var elem = document.createElement('div');
```

Comment:
It seems we do in some places in pathfora, but it would be nice to be consistent. Empty line after variable declarations, yes ... or no?

Reply:
Yeah, I am super sorry this is inconsistent. I kind of have my own style on this, (which should not be the standard for pathfora, but just to explain) I usually don't have a line break if the next line I am making some change to the variable I just defined. This is dumb though, we should standardize it and enforce it with lint. (whether that's declaring all vars at the top of the function, or just enforcing a line break after all definitions). I will include this on the upcoming pathfora roadmap ticket.

Reply:
👍